src/main/java/org/ambraproject/wombat/service/CommentServiceImpl.java

```diff
@@ -128,6 +145,28 @@ private void addCreatorData(Collection<Map<String, Object>> rootComments) {
     }));
   }
 
+  private void addArticleTitle(Collection<Map<String, Object>> rootComments) {
```

Comment:
Since this is `private` and it never uses any instance variables, you can probably make it `static` too.

Reply:
Cannot be made static as it uses to this::requestArticleTitle